### DISCUSSION TITLE
Clarification on "copilot svc ls" command

### DISCUSSION THREAD

**Discussion by tsingh2k15** [aws/copilot-cli#3150] [Q&A] [answered]
I was using "copilot svc ls" command and noticed as scenario which may demand enhancement to "copilot svc ls" command. As per docs - https://aws.github.io/copilot-cli/docs/commands/svc-ls/, currently I can only specify the application name as a filter. In my scenario, I have two environments in a given AWS account. I have one service which I am deploying to two environments in the same AWS account.  I would like to filter "copilot svc ls" output based on "environment" apart from current support of filtering by "application name".

Can this be supported?

**Comment by huanjani** [comment, marked as answer]
Hello, @tsingh2k15! 
If you do `copilot env show`, the output lists the services within the chosen environment. Does that help?
Thanks!

**Reply by tsingh2k15** [reply]
Yes,  thank you @huanjani for sharing the alternative.
